default.nix

```diff
@@ -86,7 +110,7 @@ stdenv.mkDerivation rec {
 
   prefixKey = "-prefix ";
 
-  # Note that Coq's Makefile.dune will force sequential setup, more
+  # Note that Coq's Makefile will force sequential setup, more
   # fine control can be gained by Nix by using dune directly on
   # coq-core and coq-stdlib.
```

Comment:
I think it is still somewhat true since make World will build everything. If you want to build a specific directory you can use dune build directly.

Flock is probably not needed I will remove it.

Reply:
I meant is it still true that it forces sequential setup?

Reply:
Well actually I don't know what this means. When you build, the Makefile just invokes Dune and Dune is responsible for all the parallelism. Which I believe is not an issue here. I will remove the comment.